public/video-ui/src/components/FormFields/DatePicker.js

```diff
@@ -57,19 +61,23 @@ function MinuteSelector({ date, onChange }) {
 }
 
 function DateSelector({ date, onChange }) {
+  const minDate = moment().toDate();
   const datePickerParams = {
     className: 'form__field',
-    selected: date,
-    minDate: moment(),
+    selected: date ? date.toDate() : null,
+    minDate,
     dateFormat: DATE_FORMAT,
-    readOnly: true,
+    readOnly: false,
```

Comment:
Is there a reason this has changed?

Reply:
Yeah, it looked like editing the date wasn't possible with readOnly being true. I couldn't work out why that was the case, so switching it to false seemed to allow the date to be edited. It would be good to confirm this with someone, I'll do that before merging this in.